phoenix-core/src/it/java/org/apache/phoenix/end2end/SystemTablesCreationOnConnectionIT.java

```diff
@@ -18,6 +18,9 @@
 package org.apache.phoenix.end2end;
```

Comment:
A lot of these diffs are auto-formatting by my IDE to be according to Phoenix coding standards for whitespace, line length, etc.

Reply:
Along with some changes to close statements, conns, etc. which were previously unclosed.